src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,85 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV1 = (samples) => {
+  const apiV1Samples = [{ samples: {} }];
+
+  const buildApiv1Files = (files) => {
+    const fileNames = [];
+    const apiV1Files = {};
+
+    Object.keys(files).forEach((key) => {
+      const fileType = files[key].sampleFileType;
+      let fileName = '';
+      if (fileType === 'features10x') {
+        fileName = 'features.tsv.gz';
+      } else if (fileType === 'barcodes10x') {
+        fileName = 'barcodes.tsv.gz';
+      } else if (fileType === 'matrix10x') {
+        fileName = 'matrix.mtx.gz';
+      }
```

Comment:
Hey sorry last question about this, now that I look at it wouldn't it be better to have it as an object? 
{[fileType]: fileName}

and then you don't need any ifs